lib/doctor/doctor.rb

```diff
@@ -18,7 +18,7 @@ def self.docker_messages
       running: "✅ Docker is running",
       not_running: <<~HEREDOC
         ❌ Docker is not running.
-        You should start it with `sudo brew services start dnsmasq`.
+        You should start it with `sudo brew services start docker`.
```

Comment:
I don't think it's possible to use `brew services` here, I think we'll just have to tell people to make sure the app is open.

Reply:
👍